compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/Types.java

```diff
@@ -4824,6 +4830,19 @@ private BType getIntersection(IntersectionContext intersectionContext, BType lhs
         return null;
     }
 
+    private Location combinePos(Location lhsPos, Location rhsPos) {
```

Comment:
Unless we remove this based on the previous comment, this should probably be something like `createEffectivePos`?

Reply:
Removed that method